mypy/test/helpers.py

```diff
@@ -1,6 +1,7 @@
 import sys
 import re
 import os
+import pytest  # type: ignore
```

Comment:
Why the `type: ignore`?

Ah, we don't have pytest in typeshed. This could be OK for now -- particularly as pytest is awfully metaprogramming-heavy and the stubs may end up with a fair number of `Any`s in them -- but we should really make some kind of stub, even if it's full of `Any`, rather than have a `type: ignore` here for long.

And any time we write a `type: ignore` we should always comment it with the reason why it's necessary.